pkg/fqdn/regexpmap/regexp_map_test.go

```diff
@@ -148,3 +160,38 @@ func (ds *RegexpMapTestSuite) BenchmarkRegexGroups(c *C) {
 		re.FindSubmatchIndex(in)
 	}
 }
+
+func RunRegexpMapBenchmark(c *C, matchAll bool) {
+	c.StopTimer()
+	iterations := 30000
+	m := NewRegexpMap()
+	matchformat := "foo%d.com."
+	lastMatch := 0
+
+	for i := 0; i < iterations; i++ {
```

Comment:
Could you add a comment explaining how we're building this? e.g. matchAll is always added and every 3 entries have the same regex